src/platform/utils.py

```diff
@@ -15,11 +15,105 @@
 from deeporigin.utils.core import _get_method
 
 
+class PlatformClients:
+    """
+    A container for all DeepOrigin platform API clients, instantiated with a token, base_url, and org_friendly_id.
+
+    Each API client is available as an attribute of this class, e.g., self.FilesApi, self.ToolsApi, etc.
+
+    Args:
+        token (str): The authentication token to use for all API clients.
+        base_url (str): The base URL of the DeepOrigin platform (e.g., 'https://os.deeporigin.io').
+        org_friendly_id (str): The organization-friendly ID to use for API requests.
+    """
+
+    def __init__(
+        self,
+        *,
+        token: str,
+        base_url: str,
+        org_friendly_id: str,
+    ) -> None:
```

Comment:
this class is a handy wrapper around all clients that we need to do anything on the platform. all clients are configured, instantiated, and ready to go